cascade/cascade.py

```diff
@@ -28,6 +28,7 @@
 import argparse
 import asyncio
 import datetime
+from enum import Enum
```

Comment:
Current style is to limit use of `from`. Instead just do `import enum` and below subclass from `enum.Enum`